api/api/models/base.py

```diff
@@ -12,3 +16,74 @@ def __iter__(self):
         for field_name in self._meta.get_fields():
             value = getattr(self, field_name, None)
             yield field_name, value
+
+
+class OpenverseMediaModel(Protocol):
+    media_type: MediaType
```

Comment:
This is unused, left over from another experimental approach I tried (one that didn't leverage metaclass kwargs and instead generated a new metaclass per-abstract related media model, or used a decorator, both of which were a little ugly or didn't work).

Anyway, it's meant to be removed.

Reply:
Could you leave a comment in the code saying it is meant to be removed?

Reply:
It needs to be removed from this PR. I'll do so when I get back around to making updates to this based on feedback for the overall approach.